src/universal/utils/draftjs/deserialize.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Deserializes a utf-8-encode JSON string into a draft-js ContentState instance.
+ *
+ * Fails "soft" by returning an empty ContentState rather than throwing an
+ * error on deserialization.
+ *
+ * @flow
+ */
+
+// $FlowFixMe
+import {convertFromRaw, ContentState} from 'draft-js';
+
+export default (serialized: string): ContentState => {
+  try {
+    return convertFromRaw(JSON.parse(serialized));
+  } catch (error) {
+    return ContentState.createFromText('');
```

Comment:
personally, i think this should be a hard failure. i'd rather see an error than quietly lose data.

Reply:
Yeah, I hear that. I can definitely change this to do so.